src/datetimepicker.directive.ts

```diff
@@ -9,7 +9,7 @@ import {
     OnChanges,
     SimpleChanges
 } from '@angular/core';
-import * as $ from 'jquery';
+declare var $: any;
```

Comment:
And how does that ensure you have jQuery in your scope?

Reply:
@atais I'm planning to do another PR for the doc, since user didn't installed jquery, bootstrap, moment and eonasdan-bootstrap-datetimepicker but installed by this plugin, user don't know they should add theses library in angular-cli.json ..So I think user need to install these plugin manually so they would know they need to add it in angular-cli.json, so the key is the doc.

Reply:
What if you don't use angular-cli?

Or in general case, do you need to manually add jquery in your app?
Was not the previous line enough to serve it to you?

Reply:
if you dont use angular-cli you should mention to include file (eg: in index.html ) lib are in node_modules/ (by npm install, because they are in dependecies section) but not used in the project unless you include theses lib in your project

`import * as $ from 'jquery';` will not use the global jquery which have datetimepicker() function

Reply:
OK thanks for clarification.